OtripleS.Web.Api/Services/Foundations/Classrooms/ClassroomService.Validations.cs

```diff
@@ -195,6 +178,22 @@ private static void ValidateAgainstStorageClassroomOnModify(Classroom inputClass
                         parameterValue: inputClassroom.UpdatedDate);
             }
         }
+        private static void Validate(params (dynamic Rule, string Parameter)[] validations)
+        {
+            var invalidClassroomException = new InvalidClassroomException();
+
+            foreach ((dynamic rule, string parameter) in validations)
+            {
+                if (rule.Condition)
+                {
+                    invalidClassroomException.UpsertDataList(
+                        key: parameter,
+                        value: rule.Message);
+                }
+            }
+
+            invalidClassroomException.ThrowIfContainsErrors();
+        }
```

Comment:
@tsatay this code is still not at the end of the file.

Reply:
@hassanhabib I change its order